js/cellGenerator.mjs

```diff
@@ -271,7 +274,12 @@ function initSingleCell(cell, extraColumnName, format = CELL_FORMAT_CHECKLIST, c
         }
 
         if(cell.icon){
-            htmlIcon.src = "images/Icon_" + cell.icon + ".png";
+            if(cell.icon == "Nirnroot" || cell.icon == "Wayshrine"){ //These have different markings.
+                htmlIcon.src = "images/Icon_" + cell.icon + "_Undiscovered.png";
+            }
+            else {
+                htmlIcon.src = "images/Icon_" + cell.icon + ".png";
+            }
```

Comment:
While you're making changes, can you add a
//TODO: make icons uniform again

Reply:
the way I did the _undiscovered for nirns/wayshrine is kinda terrible because of stuff like this